src/server/test/web/readingsBarGroupQuantity.js

```diff
@@ -201,8 +200,68 @@ mocha.describe('readings API', () => {
                     expectReadingToEqualExpected(res, expected, GROUP_ID);
                 });
 
-                // Add BG11 here
-
+                mocha.it('BG11: 1 day bars for 15 + 20 minute reading intervals and quantity units with +-inf start/end time & kWh as BTU reverse conversion', async () => {
+                    //load data into database
+                    const unitData = unitDatakWh.concat([
+                        {
+                            // u3
+                            name: 'MJ',
+                            identifier: 'megaJoules',
+                            unitRepresent: Unit.unitRepresentType.QUANTITY,
+                            secInRate: 3600,
+                            typeOfUnit: Unit.unitType.UNIT,
+                            suffix: '',
+                            displayable: Unit.displayableType.ALL,
+                            preferredDisplay: false,
+                            note: 'MJ'
+                        },
+                        {
+                            // u16
+                            name: 'BTU',
+                            identifier: '',
+                            unitRepresent: Unit.unitRepresentType.QUANTITY,
+                            secInRate: 3600,
+                            typeOfUnit: Unit.unitType.UNIT,
+                            suffix: '',
+                            displayable: Unit.displayableType.ALL,
+                            preferredDisplay: true,
+                            note: 'OED created standard unit'
+                        }
+                    ]);
+                    const conversionData = conversionDatakWh.concat([
+                        {
+                            // c2
```

Comment:
While this runs correctly, the design doc specifies c6 (not c2) so it forces the conversion to go the other way. That is why the description has reverse conversion.